HW_1/task_3.py

```diff
@@ -19,3 +19,22 @@
 0
 Точка на отрицательной части оси абсцисс
 """
+
+x = int(input('Введите число x:'))
+y = int(input('Введите число y:'))
+
+
+if x == 0 and y == 0:
+    print('Неверные координаты точек ;)')
+elif x > 0 and y > 0:
```

Comment:
Есть ли в условии присутствует повторяющееся значение, оператор `and`, операторы равенства или сравнения, то условие можно свернуть относительно повторяющегося значения:
```suggestion
elif x > 0 < y:
```